layouts/partials/social-buttons.html

```diff
@@ -1,4 +1,10 @@
 <div class="buttons">
+  <a class="button is-small is-github-white" target="_blank" href="{{ site.Params.github }}">
```

Comment:
Use `is-white` instead of `is-github-white`. Also, I'd recommend using `is-outlined` as well so that it looks like a button.

If you use `is-white`, you can eliminate the change that I pointed to in `style.sass`.

Reply:
Awesome, will make those changes

Reply:
When using `is-white` the logo became white, not the background. So I'll use `is-black`, and added the `is-outlined`.